rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, person
```

Comment:
- [ ] Kindly consider prefixing person with colon to make the syntax right like `:person`.